orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -122,32 +121,13 @@ class GetCommitsTask implements DiffTask {
       }
 
       return TaskResult.builder(ExecutionStatus.SUCCEEDED).context(outputs).build()
-    } catch (RetrofitError e) {
```

Comment:
There's a call to [getRepoInfo](https://github.com/spinnaker/orca/blob/b013bc8ec4de2356e0337f6ee57adacbf9776c71/orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy#L245) inside this try/catch block.  getRepoInfo uses a Front50Service bean to make a retrofit call.. Currently, that [Front50Service bean](https://github.com/spinnaker/orca/blob/b013bc8ec4de2356e0337f6ee57adacbf9776c71/orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/config/Front50Configuration.groovy#L73) doesn't use SpinnakerRetrofitErrorHandler, so I believe the code to catch RetrofitErrors needs to stay.

Reply:
I too have verified the code.. this Front50Service bean doesnt use SpinnakerRetrofitErrorHandler , so this PR is no more relevant and hence I am closing this PR